src/ngraph/op/util/binary_elementwise_logical.hpp

```diff
@@ -46,6 +46,15 @@ namespace ngraph
             class BinaryElementwiseLogical : public Op
             {
             public:
```

Comment:
Probably want to change this one to `protected`, like you did in `BinaryElementwiseArithmetic`.

Reply:
done.